agent/client/const.go

```diff
@@ -13,17 +13,22 @@ const defaultAddr = "0.0.0.0:8080"
 const apiBasePath = ""
 
 const (
-	gatewayInferControlPlanePath             = "/system/inferences"
-	gatewayInferScaleControlPath             = "/system/scale-inference"
-	gatewayInferInstanceControlPlanePath     = "/system/inference/%s/instances"
-	gatewayInferInstanceExecControlPlanePath = "/system/inference/%s/instance/%s/exec"
-	gatewayServerControlPlanePath            = "/system/servers"
-	gatewayServerLabelCreateControlPlanePath = "/system/server/%s/labels"
-	gatewayServerNodeDeleteControlPlanePath  = "/system/server/%s/delete"
-	gatewayNamespaceControlPlanePath         = "/system/namespaces"
-	gatewayBuildControlPlanePath             = "/system/build"
-	gatewayBuildInstanceControlPlanePath     = "/system/build/%s"
-	gatewayImageCacheControlPlanePath        = "/system/image-cache"
+	gatewayInferControlPlanePath                 = "/system/inferences"
+	gatewayInferScaleControlPath                 = "/system/scale-inference"
+	gatewayInferInstanceControlPlanePath         = "/system/inference/%s/instances"
+	gatewayInferInstanceExecControlPlanePath     = "/system/inference/%s/instance/%s/exec"
+	gatewayServerControlPlanePath                = "/system/servers"
+	gatewayServerLabelCreateControlPlanePath     = "/system/server/%s/labels"
+	gatewayServerNodeDeleteControlPlanePath      = "/system/server/%s/delete"
+	gatewayNamespaceControlPlanePath             = "/system/namespaces"
+	gatewayBuildControlPlanePath                 = "/system/build"
+	gatewayBuildInstanceControlPlanePath         = "/system/build/%s"
+	gatewayImageCacheControlPlanePath            = "/system/image-cache"
+	modelzCloudClusterControlPlanePath           = "/api/v1/users/%s/clusters/%s"
+	modelzCloudClusterWithUserControlPlanePath   = "/api/v1/users/%s/clusters"
+	modelzCloudClusterAPIKeyControlPlanePath     = "/api/v1/users/%s/clusters/%s/api_keys"
+	modelzCloudClusterNamespaceControlPlanePath  = "/api/v1/users/%s/clusters/%s/namespaces"
+	modelzCloudClusterDeploymentControlPlanePath = "/api/v1/users/%s/clusters/%s/deployments/%s/agent"
```

Comment:
It seems we have the client `version` and `basePath`, shall we use them?

Reply:
I think we need to use them to access Apiserver.

Reply:
I mean the client has some attribute for the prefix like `/api/v1` but may not be used now.

Reply:
I got you. I think we can consider adding the client version later.